lib/block_view_message.go

```diff
@@ -887,19 +774,14 @@ func (bav *UtxoView) _connectMessagingGroup(
 
 	// We now have a valid messaging public key, key name, and owner public key.
 	// The hard-coded default key is only intended to be registered by the owner, so we will require a signature.
-	//
-	// Note that we decided to relax this constraint after the fork height. Why? Because keeping it would have
-	// required users to go through two confirmations when approving a key with MetaMask vs just one.
-	if blockHeight < bav.Params.ForkHeights.DeSoUnlimitedDerivedKeysAndMessagesMutingAndMembershipIndexBlockHeight {
-		if EqualGroupKeyName(NewGroupKeyName(txMeta.MessagingGroupKeyName), DefaultGroupKeyName()) {
-			// Verify the GroupOwnerSignature. it should be signature( messagingPublicKey || messagingKeyName )
-			// We need to make sure the default messaging key was authorized by the master public key.
-			// All other keys can be registered by derived keys.
-			bytes := append(txMeta.MessagingPublicKey, txMeta.MessagingGroupKeyName...)
-			if err := _verifyBytesSignature(txn.PublicKey, bytes, txMeta.GroupOwnerSignature, blockHeight, bav.Params); err != nil {
-				return 0, 0, nil, errors.Wrapf(err, "_connectMessagingGroup: "+
-					"Problem verifying signature bytes, error: %v", RuleErrorMessagingSignatureInvalid)
-			}
+	if EqualGroupKeyName(NewGroupKeyName(txMeta.MessagingGroupKeyName), DefaultGroupKeyName()) {
```

Comment:
I think we want to only do this check if we're before `DeSoUnlimitedDerivedKeysBlockHeight`